### DISCUSSION TITLE
Is this project staying?

### DISCUSSION THREAD

**Discussion by ghost** [GriefPrevention/GriefPrevention#751] [Q&A] [answered]
So, when I Googled this repo, I found https://github.com/MinecraftPortCentral/GriefPrevention. Mistakenly I believed it was the official GriefPrevention repo until I investigated further. I for a minute believed that GriefPrevention died and was replaced by a $15 plugin called GriefDefender. However, I suspect something fishy is going on, so I wanted to confirm here whether or not this plugin is continuing on or being replaced.

![Screen Shot 2020-03-14 at 5 53 52 PM](https://user-images.githubusercontent.com/6305346/76691064-0ad46d80-661d-11ea-8a0b-b2702d12959a.png)

**Comment by RoboMWM** [comment]
commit, issue, and thread activity suggests I only have plans to continue and eventually refactor this plugin.

**Comment by RoboMWM** [comment, marked as answer]
He means that the sponge GP has been replaced. Bukkit/Spigot GriefPrevention is the original plugin, GP for sponge is a fork that has much deviated in design. GriefDefender does **not** replace GP. I wish he would mention "sponge" in his description.
